QuoteEngine/README.md

```diff
@@ -1,30 +1,177 @@
-Quick start, because I can't be arsed writing a full one ;)
+**Hello!**
 
-SQL setup
----------
 
-1. Create a user and database in your mysql server for the bot to use
-2. Populate the database using the .sql script
+Steps you need to do (or packages you should have) before downloading the script:
+1. Install all necessary packages for eggdrop to make use of mysql (also packages which you need to have are included).
+   <br>
+   For example for ubuntu 18.04 these would be the packages: tcl, tcl-dev, tcllib, tcl-tls, zlibc, zlib1g, tcl-trf, mysqltcl
```

Comment:
Maybe be more explicit and call it out as something like, "On ubuntu you can do `apt install tcl tcl-dev tcllib tcl-tls zlibc zlib1g tcl-trf mysqltcl`"

Reply:
i really don't want to!
if a user cannot install packages, should not install a database and mess with it, or compile things.
i also didn't say how to get eggdrop, or how to configure it. it's basics. if they don't know how to install packages with apt, this tcl script is not for them. :| sorry, but i don't think it's a good idea to make a step-by-step to install a bot, configure it, etc..